src/MSONParameterParser.h

```diff
@@ -178,6 +181,8 @@ namespace snowcrash {
                     }
                     else {
                         if (RegexMatch(attributes[i], "^enum\\[[^][]+]$")) {
+
+                            // Retrieve the type which is wrapped by enum[]
                             out.node.type = attributes[i].substr(5, attributes[i].length() - 6);
```

Comment:
> Regarding constants, I don't agree with that

There is nothing to agree or disagree here. These are magical numbers without any obvious relation to the problem. Please refactor them into constants. See "Accepted limited use of magic numbers" in http://en.wikipedia.org/wiki/Magic_number_(programming) and / or http://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Using an arbitrary numbers like this in the hope that it is obvious that some other constant somewhere is related to this is really against readability, clarity and maintainability.